ds_judgements_public_ui/templates/pages/home.html

```diff
@@ -8,107 +8,50 @@
   <meta name="robots" content="nosnippet" />
 {% endblock robots %}
 {% block header %}
```

Comment:
As far as I can tell this `header` block is now _exactly_ the same as the one in `base.html`, so we should be able to remove it completely and simplify things.